TESTING.md

```diff
@@ -0,0 +1,76 @@
+# Testing Guide for visualsign-parser
+
+For chain-specific testing approaches, see:
+- [Solana Testing Guide](src/chain_parsers/visualsign-solana/TESTING.md) - Fixture-based testing with real transaction data
+
+## Test Coverage
+
+### Prerequisites
+
+```bash
+# Install cargo-llvm-cov
+cargo install cargo-llvm-cov
+```
+
+### Viewing Coverage
+
+```bash
+# Generate HTML report and open in browser (simplest approach)
+cargo llvm-cov --workspace --open
+
+# Or for a specific package
+cargo llvm-cov --package visualsign-solana --lib --open
+```
+
+Reports are generated in `target/llvm-cov/html/` and opened automatically.
```

Comment:
This is enough for people on remote workstations. They know how to open html files. You can skip the rest.